### PR TITLE
Fix the bug of pom display

### PR DESCRIPTION
**Pull request by teeyog**
Problem Summary:

### What is changed and how it works?

Proposal: [xxx](url) 

Feel free to ping committers for the review!

## Contribution Checklist

  - Make sure that the pull request corresponds to a [GITHUB issue](https://github.com/streamxhub/streamx/issues).

  - Name the pull request in the form "[Feature] [component] Title of the pull request", where *Feature* can be replaced by `Hotfix`, `Bug`, etc.

  - Minor fixes should be named following this pattern: `[hotfix] [docs] Fix typo in README.md doc`.

-->

### CODE REVIEW

File: streamx-console/streamx-console-webapp/src/views/flink/app/Add.vue [hunk 1]
```diff
@@ -1997,7 +1997,7 @@
       handleEditPom(pom) {
         const pomString = toPomString(pom)
         this.activeTab = 'pom'
```
Comment (wolfboys) [inline streamx-console/streamx-console-webapp/src/views/flink/app/Add.vue]: There is also have bug in line 1999, it should be changed to ` this.controller.activeTab`  you can fix it together

**Comment by wolfboys** [comment]
This is indeed a bug, thanks for your contribution

File: streamx-console/streamx-console-webapp/src/views/flink/app/Add.vue (hunk 1 above)
Comment (teeyog) [reply streamx-console/streamx-console-webapp/src/views/flink/app/Add.vue]: done

**Comment by wolfboys** [comment]
LGTM

**Pull request merged**